src/renderer/components/overlays/pref-overlay/entries-pref/read-only-pref/ReadOnlyPref.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from "react";
+
+import { translations } from "../../../../../utils/i18n";
+
+export interface StateProps {
+	readOnly: boolean;
+}
+
+export interface DispatchProps {
+	updateReadOnlyPref: (readOnly: boolean) => void;
+}
+
+type Props = StateProps & DispatchProps;
+
+/**
+ * Preference fieldset for setting the editor state
```

Comment:
Probably will want to update this default comment

Reply:
Is it ok if I put it like this: Preference fieldset for setting the editor in read only mode?